pkg/backup/item_backupper.go

```diff
@@ -415,9 +420,14 @@ func (ib *defaultItemBackupper) takePVSnapshot(obj runtime.Unstructured, log log
 		}
 	}
 
-	pvFailureDomainZone := pv.Labels[zoneLabel]
-	if pvFailureDomainZone == "" {
-		log.Infof("label %q is not present on PersistentVolume", zoneLabel)
+	// TODO: prefer stable labels for region in >= v1.18
```

Comment:
Let's be a little more explicit with this `// TODO` -- something like "once failure-domain.beta.kubernetes.io/zone is no longer supported in any velero-supported version of Kubernetes, remove fallback checking of it"